osbot_browser/chrome/Chrome_Setup.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, chrome_args, options):
         self._browser         : Browser     = None
         self.chrome_args      : Chrome_Args = chrome_args
         self.options          : dict        = options
-        self.s3_chrome_binary : tuple       = ('gw-bot-lambdas','lambdas-dependencies/chromium-2_1_1')
+        self.s3_chrome_binary : tuple       = (os.environ.get('OSBOT_LAMBDA_S3_BUCKET' , 'gw-bot-lambdas') ,'lambdas-dependencies/chromium-2_1_1')
```

Comment:
@dinis-cruz-gw Would this cause any issue for other deployment of the lambda?? I have reused the variable name

Reply:
yes this should be an environment variable

Reply:
yes this should be an environment variable